tests/integration/checkpoint.bats

```diff
@@ -159,10 +159,13 @@ function simple_cr() {
 	# checkpoint the running container
 	mkdir image-dir
 	mkdir work-dir
-	runc --criu "$CRIU" checkpoint --parent-path ./parent-dir --work-path ./work-dir --image-path ./image-dir test_busybox
+	runc --criu "$CRIU" checkpoint --parent-path ../parent-dir --work-path ./work-dir --image-path ./image-dir test_busybox
```

Comment:
So what I still don't understand is why the test was not failing.

Reply:
I think that is CRIU's fault and confuses many people when using pre-dump. CRIU silently switched to non pre-dump if `--parent-path` is not correct. https://github.com/checkpoint-restore/criu/pull/1403

Restore also works without any message, but it is not using the pre-dump. This will probably be an error in the next CRIU release. Having it fixed in runc already know makes sense.